tests/test_backend_cache.py

```diff
@@ -226,14 +226,14 @@ def make_cache(self, request):
         else:
             host = redis.Redis()
 
-        c = backends.RedisCache(host=host, key_prefix="werkzeug-test-case:")
+        c = backends.RedisCache(host=host, key_prefix=lambda: "werkzeug-test-case:")
```

Comment:
Please add another test case which tests the key_prefix as a function instead of changing this test case.